### ISSUE TITLE
fix docs formatting

### ISSUE THREAD

**Issue by Borda** [PyTorchLightning/pytorch-lightning#717]
## 📚 Documentation

At this moment the docs raise about 140 warnings which may cause some issues in documentation.
To build the docs locally follow these steps:
```bash
cd docs
pip install -r requirements.txt
make html
```

Thanks!

**Comment by hanbyul-kim** [comment]
Nice to meet you! I'm Hanbyul Kim, a ML engineer working in South Korea.
I'd like to contribute to ML open source for the first time, so I've been looking around several projects lately. And I found your repo and felt it's cool. 

Since I'm a newcomer, I guess it can be a good idea to start doing some "good first issue" things. I'm polishing your code to remove warnings while generating docs. I've just pushed my branch right now, but I think I need more time to do all the things. Is it ok for me to do this job?
